api/src/main/java/keywhiz/api/model/Client.java

```diff
@@ -72,7 +72,7 @@ public class Client {
   private final boolean enabled;
 
   @JsonProperty
-  private final String owner;
+  private String owner;
```

Comment:
Same as the last PR: making the owner field writable simplifies the DAO layer code.